libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/model/RouteLineColorResources.kt

```diff
@@ -604,31 +702,38 @@ class RouteLineColorResources private constructor(
-                alternativeRouteUnknownCongestionColor,
-                restrictedRoadColor,
-                routeClosureColor,
-                alternativeRouteRestrictedRoadColor,
-                alternativeRouteClosureColor,
-                routeLineTraveledColor,
-                routeLineTraveledCasingColor,
-                routeCasingColor,
-                alternativeRouteCasingColor,
-                inActiveRouteLegsColor
+                lowCongestionRange = lowCongestionRange,
+                moderateCongestionRange = moderateCongestionRange,
+                heavyCongestionRange = heavyCongestionRange,
+                severeCongestionRange = severeCongestionRange,
+                routeDefaultColor = routeDefaultColor,
+                routeLowCongestionColor = routeLowCongestionColor,
+                routeModerateCongestionColor = routeModerateCongestionColor,
+                routeHeavyCongestionColor = routeHeavyCongestionColor,
+                routeSevereCongestionColor = routeSevereCongestionColor,
+                routeUnknownCongestionColor = routeUnknownCongestionColor,
+                inactiveRouteLegLowCongestionColor = inactiveRouteLegLowCongestionColor,
+                inactiveRouteLegModerateCongestionColor = inactiveRouteLegModerateCongestionColor,
+                inactiveRouteLegHeavyCongestionColor = inactiveRouteLegHeavyCongestionColor,
+                inactiveRouteLegSevereCongestionColor = inactiveRouteLegSevereCongestionColor,
+                inactiveRouteLegUnknownCongestionColor = inactiveRouteLegUnknownCongestionColor,
+                alternativeRouteDefaultColor = alternativeRouteDefaultColor,
+                alternativeRouteLowCongestionColor = alternativeRouteLowCongestionColor,
+                alternativeRouteModerateCongestionColor = alternativeRouteModerateCongestionColor,
+                alternativeRouteHeavyCongestionColor = alternativeRouteHeavyCongestionColor,
+                alternativeRouteSevereCongestionColor = alternativeRouteSevereCongestionColor,
+                alternativeRouteUnknownCongestionColor = alternativeRouteUnknownCongestionColor,
+                restrictedRoadColor = restrictedRoadColor,
+                routeClosureColor = routeClosureColor,
+                inactiveRouteLegRestrictedRoadColor = inactiveRouteLegRestrictedRoadColor,
+                inactiveRouteLegClosureColor = inactiveRouteLegClosureColor,
+                alternativeRouteRestrictedRoadColor = alternativeRouteRestrictedRoadColor,
+                alternativeRouteClosureColor = alternativeRouteClosureColor,
+                routeLineTraveledColor = routeLineTraveledColor,
+                routeLineTraveledCasingColor = routeLineTraveledCasingColor,
+                routeCasingColor = routeCasingColor,
+                alternativeRouteCasingColor = alternativeRouteCasingColor,
+                inActiveRouteLegsColor = inActiveRouteLegsColor,
```

Comment:
Now this one only makes sense in case traffic is disabled, right?

Reply:
Could you elaborate?

Reply:
Otherwise this colour won't be visible?

Reply:
When traffic is enabled and unknown congestion is made transparent (which should be the default and I don't understand why it isn't) the inactive styling has an effect, so it's valid regardless of whether traffic is enabled or not.

Reply:
OK.